types/ldapjs-client/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": false
```

Comment:
Why is this needed?

Reply:
I wrote it in the PR description, to be more verbose, it gives the following error:
```
npm-naming  The types for ldapjs-client specify 'export default' but the source does not mention 'default' anywhere.
```

The sources of ldapjs-client have `module.exports = Client;`, but I have to use `export default LdapClient` on the typings